apps/src/blockly/addons/contextMenu.js

```diff
@@ -0,0 +1,179 @@
+import GoogleBlockly from 'blockly/core';
+
+const registerDeletable = function() {
+  /** @type {GoogleBlockly.ContextMenuRegistry.RegistryItem} */
```

Comment:
I think we should go ahead and remove these

Reply:
I'll be at a classroom visit tomorrow morning, so gonna approve now since removing these comments isn't necessarily a blocker!